### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by vikas924**
Hello there,

In this project, I have set up a "Portfolio-setup-and-mobilefirst" repository. My tasks include:

1.Documented a professional README.md.
2.Included a .gitignore file in main branch.
3.Created a new branch according to the GitHub flow rules.
4.Added files in new branch with seperate commits.
5.Added html file that includes contact section for portfolio mobile first website.
6.Added css file that include styles for contact section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -154,7 +154,48 @@ <h3 class="skills-head"><img class="skill-bullets" src="./Work-section-img/squar
           </div>
         </div>
         </div>
+        </div>
+         
+        <div class="contact">
+            <div class="contact-grid">
+            <div class="contactpara">
+                <p class="contactp">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
```
Comment (AdnanOlarmmi) [inline index.html:162]: - [x] Please, ensure that this text follows the template design guideline to achieve almost a 💯 pixel-perfect design 😉. Things like the line height, font size, weight et all. 😀 

- Your design
wrong2

- Figma design 🙃
right2

File: index.html [hunk 2]
```diff
@@ -154,7 +154,48 @@ <h3 class="skills-head"><img class="skill-bullets" src="./Work-section-img/squar
...
+                    </div>
+                    <div class="email">
+                    <input type="email" name="Email-address" placeholder="email address" class="email1" required>
+                    <div class="blank2"></div>
+                    </div>
+                    <div class="text">
+                    <textarea name="comment" placeholder="Enter text here" class="text1" maxlength="500" required></textarea>
+                    </div>
+                    <button name="submit" class="submit">Get in touch</button>
+                    </form>
```
Comment (AdnanOlarmmi) [inline index.html:178]: - [ ] Kindly let's stick to the Figma design as much as possible for the Form also 😀.

- Your design
wrong

- Figma design
right

😀. By giving padding to the input elements more, ensuring the placeholders are the exact ones in the Figma design in terms of the spacing, the casing(upper or lower case and the likes), and likewise removing the `border` or `outline` properties of the input elements but on `focus` and off `focus`, please check [this](https://www.w3schools.com/cssref/sel_focus.php) for more info. 😉

File: index.html [hunk 3]
```diff
@@ -154,7 +154,48 @@ <h3 class="skills-head"><img class="skill-bullets" src="./Work-section-img/squar
...
+                    <div class="blank1"></div>
+                    </div>
+                    <div class="email">
+                    <input type="email" name="Email-address" placeholder="email address" class="email1" required>
+                    <div class="blank2"></div>
+                    </div>
+                    <div class="text">
+                    <textarea name="comment" placeholder="Enter text here" class="text1" maxlength="500" required></textarea>
+                    </div>
+                    <button name="submit" class="submit">Get in touch</button>
```
Comment (AdnanOlarmmi) [inline index.html:177]: - [x] Nice work with the hover properties 🙂, However somethings are still left in the hover design 😀

- Your design 
wrong1

- Figma design
right1

Did you spot the difference? 🌚.

Things like the border and the "Get in touch" text size and weight.

**Review by AdnanOlarmmi — CHANGES_REQUESTED** [review]
Hi @team, 

This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

You're Almost there, funny one eyed alien

## Highlights
- Professional Readme.
- Nice one working in pairs.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — APPROVED** [review]
Hi @vikas924,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Form tag ✔️ 
- input attributes ✔️ 
- correct use of formspree ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**